### PR TITLE
[Fix] DLCs list on Install Dialog

### LINKED ISSUE #891: Install button won\t show when DLC list is too long
### Try to get help on our Wiki first: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Troubleshooting

**Describe the bug**

-----------------------------------------------------------------------------------------------------------------------------------------------
I tried to install the game HITMAN3, but the installation interface shows that the installed game content is too long, so the install button cannot be displayed, and I cannot install this game！！！！！！！！！！！！！！！
-----------------------------------------------------------------------------------------------------------------------------------------------

**Add Logs**

Check on how to [run Heroic from Terminal or Powershell](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Troubleshooting#running-heroic-from-terminalpowershell-to-debug-errors) and put some logs here

**To Reproduce**

**Expected behavior**

**Screenshots**
Problem

**Desktop (please complete the following information):**

Version Windows 10 Pro
Version number 21H1
Installation Date ‎2021/‎10/‎15
OS Build 19043.1288
Experience Windows Feature Experience Pack 120.2212.3920.0

- Heroic Version [e.g. 22] 2.0.2
- Game:HITMAN3
- Wine/Proton version: 
- Hardware specs (RAM, CPU, GPU):
- CPU: Inter Core i7 7700HQ 
- GPU: Geforce GTX  1050TI
- RAM: SAMSUNG 24GB 2400Mhz

**Additional context**
Can you add an install button？Thank you very much!!!!!!!!!!!!!!!!!!

### PR DESCRIPTION
**Pull request by flavioislima**
Fixes #891 

Also removed DLC list from Game page since it is also shown on the Install Dialog

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/screens/Library/components/InstallModal/index.css [hunk 1]
```diff
@@ -82,6 +82,12 @@
   margin: 12px;
 }
 
+.dlcList {
+  padding: 12px 0 18px;
+  display: flex;
+  flex-direction: column;
```
Comment (arielj) [inline src/screens/Library/components/InstallModal/index.css:88]: Maybe we need a max-height here with a scroll? not sure but I imagine some games have a lot of DLCs (which causes the original issue) making the install modal too tall maybe even overflow the screen.

**Review by arielj — COMMENTED** [review]
hey @flavioislima, just added a comment with a concern I have on the size of the DLC list element, otherwise it looks good to me

File: src/screens/Library/components/InstallModal/index.css (hunk 1 above)
Comment (flavioislima) [reply src/screens/Library/components/InstallModal/index.css:88]: yes, makes sense. I'll change it 👍🏽

Comment (flavioislima) [reply src/screens/Library/components/InstallModal/index.css:90]: done

**Review by Nocccer — APPROVED** [review]

**Pull request merged**